scripts/config_template.yaml

```
@@ -25,7 +25,25 @@ forecast_timeframe:
 evaluation_timeframe:
   interval: 14d # Leave this blank if you don't want evaluation
 
-models: [LinearIncidentUptakeModel]
+# Details of the models to fit
+models:
+  [{name: LinearIncidentUptakeModel,
+    seed: 0,
+    params:
+      {a_mn: 0.0,
+      a_sd: 1.0,
+      bP_mn: 0.0,
+      bP_sd: 1.0,
+      bE_mn: 0.0,
+      bE_sd: 1.0,
+      bPE_mn: 0.0,
+      bPE_sd: 1.0,
+      sig_mn: 1.0},
+    mcmc:
+      {num_warmup: 1000,
+      num_samples: 1000,
+      num_chains: 4}
+  }]
```

```suggestion
  - name: LinearIncidentUptakeModel
    seed: 0
    params:
      a_mn: 0.0
      a_sd: 1.0
      bP_mn: 0.0
      bP_sd: 1.0
      bE_mn: 0.0
      bE_sd: 1.0
      bPE_mn: 0.0
      bPE_sd: 1.0
      sig_mn: 1.0
    mcmc:
      num_warmup: 1000
      num_samples: 1000
      num_chains: 4
```
In yaml, you can use the `-` to indicate that that's a list item, but then the list item can just be a dictionary (with nested dictionaries)

---

I like this! I didn't understand what it was the first time I saw it. I will make the change.